source/fem-dem/cfd_dem_coupling.cc

```diff
@@ -156,11 +172,32 @@ CFDDEMSolver<dim>::read_dem()
 
 template <int dim>
 inline bool
-CFDDEMSolver<dim>::check_contact_search_step_constant(unsigned int counter)
+CFDDEMSolver<dim>::check_contact_search_step_constant(
+  const unsigned int &counter)
 {
   return ((counter % contact_detection_frequency) == 0);
 }
 
+template <int dim>
+inline bool
+CFDDEMSolver<dim>::check_contact_search_step_dynamic(const unsigned int &)
+{
+  bool sorting_in_subdomains_step =
```

Comment:
Add comments or documentation here, this is not clear

Reply:
done